docs/installing_deis/aws.rst

```diff
@@ -80,7 +80,7 @@ Customize cloudformation.json
 -----------------------------
 
 Any of the parameter defaults defined in deis.template.json can be overridden by setting the value
-in `cloudformation.json`_ like so:
+in `cloudformation.json`_ - for example, to configure all of the optional settings:
```

Comment:
I think this should be an em-dash `—` or better, a new sentence starting "For example,".